lib/worker_tools.rb

```diff
@@ -1,4 +1,4 @@
-Dir[File.join(__dir__, 'worker_tools/**/*.rb')].each { |path| require path }
+Dir[File.join(__dir__, 'worker_tools/**/*.rb')].sort.each { |path| require path }
```

Comment:
I followed this recommendation 

[Don't require files in random order](https://makandracards.com/makandra/47551-don-t-require-files-in-random-order)

> A common pattern in Ruby is to to require all files in a specific diretory, using something like `Dir.glob(...)`
> However, this causes files to be required in an order determined by the file system. Since load order can be important, this may lead to different behavior on different machines which are hard to debug.

with ruby 3 it would not be longer needed